save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityGeneralInfo.kt

```diff
@@ -63,14 +89,54 @@ val vulnerabilityGeneralInfo: FC<VulnerabilityGeneralInfo> = FC { props ->
                     }
                 }
                 hr { }
-                h6 {
-                    className = ClassName("font-weight-bold text-primary mb-4")
-                    +"Description"
+                div {
+                    className = ClassName("row")
+                    h6 {
+                        className = ClassName("ml-3 font-weight-bold text-primary mb-4")
+                        +"Description"
+                    }
+                    if (true) {
+                        ReactHTML.button {
+                            type = ButtonType.button
+                            className = ClassName("btn btn-link text-xs text-muted text-left ml-auto")
+                            +"Edit  "
+                            fontAwesomeIcon(icon = faEdit)
+                            onClick = {
+                                setIsEditDisabled(false)
+                            }
+                        }
+                    }
                 }
-                textarea {
-                    className = ClassName("auto_height form-control-plaintext pt-0 pb-0")
-                    value = description
-                    rows = 8
+                div {
+                    textarea {
+                        className = ClassName("auto_height form-control-plaintext pt-0 pb-0")
+                        value = description
+                        disabled = isEditDisabled
+                        rows = 8
+                        onChange = {
+                            setDescription(it.target.value)
+                        }
+                    }
+                    button {
```

Comment:
Why not `buttonBuilder`? Why don't you use it?

Reply:
ctrl+c / ctrl+v is evil :)
Fixed it